core-web/libs/template-builder/src/lib/components/template-builder/components/remove-confirm-dialog/remove-confirm-dialog.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Output } from '@angular/core';
+
+import { ConfirmationService } from 'primeng/api';
+import { ButtonModule } from 'primeng/button';
+import { ConfirmPopupModule } from 'primeng/confirmpopup';
+
+@Component({
+    selector: 'dotcms-remove-confirm-dialog',
+    templateUrl: './remove-confirm-dialog.component.html',
+    styleUrls: ['./remove-confirm-dialog.component.scss'],
+    standalone: true,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    imports: [ConfirmPopupModule, ButtonModule],
+    providers: [ConfirmationService]
+})
+export class RemoveConfirmDialogComponent {
+    @Output() deleteConfirmed: EventEmitter<void> = new EventEmitter();
+    @Output() deleteRejected: EventEmitter<void> = new EventEmitter();
+    constructor(private confirmationService: ConfirmationService) {}
+
+    openConfirmationDialog(event: Event): void {
+        this.confirmationService.confirm({
+            target: event.target,
+            message: 'Are you sure you want to proceed deleting this item?',
```

Comment:
After we merge @rjvelazco PR we need to update this.

Reply:
Also we might need in the future to pass this message because we might want to custom this message for diff items, like: "Are you sure you want to delete this box?".